### ISSUE TITLE
Bag search not working on battle pets

### ISSUE THREAD

**Issue by zeenk** [Jaliborc/Bagnon#1685]
Was trying to find my battle pet Steward Featherling in my super messy bags that I haven't bothered to clean up yet but noticed when I tried to find it with search it didn't get highlighted.

![2023-05-18_06-48-07](https://github.com/Jaliborc/Bagnon/assets/68170439/cbb143ce-0f35-40fe-b7fb-9b381d496257)

**Comment by Fleymour** [comment]
yeah had yesterday search not working too.. (reload helped).. maybe this is connected?

1x BagBrother/libs/ItemSearch-1.3-4/API.lua:107: script ran too long
[string "@BagBrother/libs/ItemSearch-1.3-4/API.lua"]:107: in function `BelongsToSet'
[string "@Scrap/addons/main/main.lua"]:164: in function `IsFiltered'
[string "@Scrap/addons/main/main.lua"]:76: in function `IsJunk'
[string "@Scrap/addons/main/main.lua"]:104: in function <Scrap/addons/main/main.lua:92>
[string "=(tail call)"]: ?
[string "@Scrap/addons/merchant/button.lua"]:144: in function `UpdateState'
[string "@Scrap/addons/merchant/button.lua"]:75: in function `?'
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:114: in function <...Ons/Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:114>
[string "=[C]"]: ?
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:24: in function <...Ons/Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:20>
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:59: in function `SendMessage'
[string "@BagBrother/libs/WildAddon-1.0-1/WildAddon-1.0.lua"]:83: in function `SendSignal'
[string "@Scrap/addons/main/main.lua"]:47: in function <Scrap/addons/main/main.lua:47>
[string "=[C]"]: ?
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:24: in function <...Ons/Ace3/CallbackHandler-1.0/CallbackHandler-1.0.lua:20>
[string "@Ace3/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:59: in function `Fire'
[string "@Ace3/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
Skipped (In Encounter)
